db/migrate/20190424094003_fix_provider_enrichment_accrediting_provider_field_change.rb

```diff
@@ -0,0 +1,17 @@
+class FixProviderEnrichmentAccreditingProviderFieldChange < ActiveRecord::Migration[5.2]
+  def self.up
+    execute "UPDATE   provider_enrichment
+                SET   json_data = Replace(json_data :: text, '\"UcasInstitutionCode\":',
+                              '\"UcasProviderCode\":') ::
+                              jsonb
```

Comment:
Turning the `jsonb` into `text` just to do a search-and-replace feels a bit risky. Is there no way to do this by using the built-in jsonb functions in PostgreSQL?

Reply:
you asking for the head bang option...

Reply:
I just looked at the structure of the data that's being changed. 🙀 😭 What a nightmare. But yes, you are right, the structure of the JSON data is a bit too complex to be able to operate on using the json functions built-into Postgres.

Reply:
:upside_down_face: